metadata_annotation.ipynb

```diff
@@ -0,0 +1,2466 @@
+{
```

Comment:
<p>So what do I see? Interpret the plot.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>